app/Services/VoucherEvaluator/Evaluations/ChildIsBetweenOneAndSecondarySchoolAge.php

```diff
@@ -6,19 +6,30 @@
 use App\Specifications\IsUnderStartDate;
 use Carbon\Carbon;
 use Chalcedonyt\Specification\AndSpec;
+use Chalcedonyt\Specification\NotSpec;
 
-class ChildIsUnderSecondarySchoolAge extends BaseChildEvaluation
+class ChildIsBetweenOneAndSecondarySchoolAge extends BaseChildEvaluation
```

Comment:
You probably don't need this as you got it covered by `ChildIsBetweenOneAndPrimarySchoolAge` and `ChildIsPrimarySchoolAge`. but like 🎅 , check it twice 👯‍♀️ 👯‍♂️

Reply:
removed class